export: Command not found.
export: Command not found.
export: Command not found.
export: Command not found.
Badly placed ()'s.
 
Running threaded PSI4 on node18 
/home/mdavis/anaconda3/envs/p4env/bin/psi4
 
Working and scratch directories
/home/mdavis/f12eom_optimizer/tests/slurm
/scratch/mdavis/PSI4_2022-08-18_11:44:08.942460617
 
use -n:  to specify number of threads
use -i:  to specify input  file     (default = input.dat)
use -o:  to specify output file     (default = file.out )
 
-h or --help for more options
 
Running...
Traceback (most recent call last):
  File "/home/mdavis/f12eom_optimizer/f12eom_optimizer/f12eom_optimizer.py", line 275, in <module>
    main()
  File "/home/mdavis/f12eom_optimizer/f12eom_optimizer/f12eom_optimizer.py", line 255, in main
    lastStepOK = history.oHistory.current_step_report()
  File "/home/mdavis/optking/optking/history.py", line 141, in current_step_report
    opt_step_report = "\n\tCurrent energy: %20.10lf\n" % self.steps[-1].E
IndexError: list index out of range
...Done
 
